ros2topic/ros2topic/verb/pub.py

```diff
@@ -131,17 +136,18 @@ def main(self, *, args):
 
 
 def main(args):
+    qos_profile = get_pub_qos_profile()
 
     qos_profile_name = args.qos_profile
-    if not qos_profile_name:
-        qos_profile_name = default_profile_str
+    if qos_profile_name:
+        qos_profile = rclpy.qos.QoSPresetProfiles.get_from_short_key(qos_profile_name)
+    profile_configure_short_keys(qos_profile, args.qos_reliability, args.qos_durability,
+                                 args.qos_depth, args.qos_history)
```

Comment:
nit:

```suggestion
    profile_configure_short_keys(
        qos_profile, args.qos_reliability, args.qos_durability,
        args.qos_depth, args.qos_history)
```

linters accept both styles, but we usually prefer this one.